### PR TITLE
Change operator default port to 443.

### PR DESCRIPTION
**Pull request by JoshVanL**
The operator service should serve on 443 as this is the more proper TLS port over the current 80. The injector has been updated to now inject port 443 to sidecars. Port 80 is still being served for backwards compatibility, but should be removed in v1.14.

### CODE REVIEW

File: charts/dapr/dapr-0.0.0.tgz
Comment (daixiang0) [inline charts/dapr/dapr-0.0.0.tgz:1]: Please remove it, if it is not in gitignore, update as well.

Comment (JoshVanL) [reply charts/dapr/dapr-0.0.0.tgz:1]: Thanks, fixed!

**Review by mukundansundar — APPROVED** [review]

**Comment by mukundansundar** [comment]
> The operator service should serve on 443 as this is the more proper TLS port over the current 80. The injector has been updated to now inject port 443 to sidecars. Port 80 is still being served for backwards compatibility, but should be removed in v1.14.

It should ideally be only removed in 1.15, 2 releases after deprecation.

**Comment by JoshVanL** [comment]
@mukundansundar It was my understanding that this was not the case for non-API related control plane configuration options. For example we do not do this for removing CLI flags.

**Review by daixiang0 — APPROVED** [review]
LGTM

**Pull request merged**